frontend/src/app/Catalog/CatalogView/CatalogView.tsx

```diff
@@ -216,48 +328,60 @@ const CatalogView = (props: CatalogViewProps) => {
 							</p>
 						</>
 					</ReadMore>
-					<h5>Class Times - {semester ?? ''}</h5>
-					{sections && sections.length > 0 ? (
-						<SectionTable sections={sections} />
-					) : !loading ? (
-						<span>There are no class times for the selected course.</span>
-					) : null}
-
-					{/*
-					Redesigned catalog sections
-					<CatalogViewSections sections={sections} />
-					*/}
-
-					{/* Good feature whenever we want...
-					<h5>Past Offerings</h5>
-					<section className={styles.pills}>
-						{pastSemesters ? (
-							pastSemesters.map((req) => (
-								<button
-									className={styles.pill}
-									key={req.id}
-									onClick={() =>
-										history.push(`/catalog/${req.name}/${course.abbreviation}/${course.courseNumber}`)
-									}
-								>
-									{req.name}
-								</button>
-							))
-						) : (
-							<Skeleton
-								style={{ marginRight: '5px' }}
-								inline
-								count={10}
-								width={80}
-								height={28}
-								borderRadius={12}
-							/>
-						)}
-					</section> */}
+					{loading ? 
+						<BTLoader></BTLoader> 
+						:
+						<div>
+						
+							{<CatalogTabs 
+								tab={tab} 
+								semester={semester} 
+								course={course} 
+								sections={sections} 
+								links={links} 
+								loading={loading} 
+								gradesGraphData={gradesGraphData} 
+								gradesData={gradesData} 
+								setTab={setTab} />}
+
```

Comment:
`tab`, `links` `gradesGraphData`, `gradesData` can all go inside the tab component

Reply:
added to tab component